src/MIDebugEngine/AD7.Impl/AD7Property.cs

```diff
@@ -430,25 +432,10 @@ public int GetDataBreakpointInfo160(out string pbstrAddress, out uint pSize, out
         {
             try
             {
-                pbstrAddress = _variableInformation.Address();
+                pbstrAddress = _variableInformation.Address() + "," + _variableInformation.Name;
                 pSize = _variableInformation.Size();
                 pbstrDisplayName = _variableInformation.Name;
```

Comment:
>pbstrDisplayName [](start = 16, length = 16)

To address:

> 1. What should the display name be - Name or FullName(), and why? (This is a good learning opportunity for me to understand why one over the other and gives me insight as to the decision process.)

The purpose of the display name is to be the name for the data breakpoint in the breakpoints window. You don't want it to be just 'Name' because, in general, 'Name' isn't very descriptive. Ex: I I drill down into 'a.b.c' the name for 'c' is just 'c' rather than 'a.b.c'. We could probably do even better than FullName() since MIEngine does some tricks with FullName to make it easier to evaluate (ex: converting pointers). But it should still be better than name.

Reply:
Thanks!